css/values.css

```diff
@@ -0,0 +1,336 @@
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+.hero {
+	background: linear-gradient(
+			260deg,
+			rgba(9, 9, 9, 0.8) 46.9%,
+			rgba(9, 9, 9, 0) 86.82%
+		),
+		url("../assets/images/pages/values/hero-values.png") top/cover;
+	display: flex;
+	align-items: center;
+}
+/* values heading layout */
+.hero .wrapper {
+	display: flex;
+	justify-content: flex-end;
+}
+/* values heading sizing */
+.hero-text-container {
+	width: 50%;
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* 
+===================================
+Values
+===================================
+*/
+
+/* layout */
+.values-list,
+.values-row,
+.values-img-text {
+	display: flex;
+	gap: var(--spacing-sm);
+}
+
+.values-list,
+.values-img-text {
+	flex-direction: column;
+}
+
+/* card widths */
+.sustainability,
+.authenticity {
+	width: 60%;
+}
+
+.humanity,
+.multigenerational {
+	width: 40%;
+}
+
+.education,
+.innovation {
+	width: 50%;
+}
+
+/* general card styling */
+.values-card {
+	padding: 60px;
+	color: var(--text-color-light);
+	border-radius: var(--border-radius);
+
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+/* text layout*/
+.values-card .text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* card specific */
+.humanity {
+	background-color: var(--color-copper);
+}
+
+.sustainability {
+	background: linear-gradient(
+			269deg,
+			#0e0e0e 46.26%,
+			rgba(27, 27, 27, 0) 99.04%
+		),
+		url("/assets/images/pages/values/sustainability.png") top/cover;
+}
+
+.authenticity {
+	background: linear-gradient(
+			88deg,
+			#b91515 47.26%,
+			rgba(185, 21, 21, 0.73) 55.57%,
+			rgba(119, 45, 45, 0.4) 68.47%
+		),
+		url("/assets/images/pages/values/authenticity.png") top/cover;
+}
+.multigenerational {
+	background-color: var(--color-grape);
+}
+.education {
+	background: linear-gradient(
+			90deg,
+			#184f85 63.23%,
+			rgba(24, 79, 133, 0) 83.39%
+		),
+		url("/assets/images/pages/values/education.png") top/cover;
+}
+
+.innovation {
+	background: linear-gradient(
+			280deg,
+			#007a80 67.16%,
+			rgba(0, 122, 128, 0) 80.36%
+		),
+		url("/assets/images/pages/values/innovation.png") top/cover;
+}
+
+/* 
+===================================
+Support Banner
+===================================
+*/
+.support {
+	background-color: var(--color-blue);
+}
+
+/* text adjustments */
+.support h2 {
+	color: var(--color-light-amaranth);
+}
+
+.support p {
+	color: var(--text-color-light);
+	font-size: 46px;
+	font-weight: 700;
+	line-height: 50px;
+}
+
+.support a {
+	width: fit-content;
+}
+
+.support-text {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-sm);
+}
+
+/* layout */
+.support .wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: var(--spacing-med);
+}
+
+/* heading and heart layout*/
+.support-heading {
```

Comment:
Limit width 

<img width="1431" alt="Screen Shot 2024-01-11 at 3 12 14 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/c9dcb811-0637-4037-9c61-9659b5b8c1ed">

Reply:
adjusted for the `p` element specifically at line `171`

![image](https://github.com/indigenousfriends/itlct-website-development/assets/98284615/077306f1-275c-4fe7-bb90-0c262d615fc8)